### ISSUE TITLE
Android Auto: Shuffle All

### ISSUE THREAD

**Issue by morph027** [timusus/Shuttle#414]
Hi!

I just stumbled upon your great app and was playing around with Android Auto in my new car. It works great besides i'm missing some ability to shuffle all music in car mode like in the normal app. Would be great to have this feature!

**Comment by timusus** [comment]
Yes, I agree this should be implemented.

**Comment by pandeiro** [comment]
Normally I try to refrain from "Me, too!"-ing but... Me, too! I would love this feature!

**Comment by techexo** [comment]
I agree that it is in my opinion a must have feature that is missing right now. It would be great to be able to read all albums from an artist (shuffled or not), and not just the one album.

I do hope it can be implemented! Thanks for the app anyway ☺️
